### ISSUE TITLE
Terraform CLI task version should not be hard-coded

### ISSUE THREAD

**Issue by AlvinStanescu** [charleszipp/azure-pipelines-tasks-terraform#63]
The most recent version of this extension (0.4.4) still has the version number for the TerraformCLI task hard-coded to `0.2.4`, instead of using GitVersion, similar to the TerraformInstaller task.

```
    "Major": "#{GitVersion.Major}#",
    "Minor": "#{GitVersion.Minor}#",
    "Patch": "#{GitVersion.Patch}#"
```

Due to version-based caching, Azure DevOps still seems to use an older version of this task for us, which did not support the `destroy` command.

**Comment by charleszipp** [comment]
Thanks for reporting this @AlvinStanescu. I am working on a PR to get this corrected.

**Comment by charleszipp** [comment]
Fixed with 0.4.5
